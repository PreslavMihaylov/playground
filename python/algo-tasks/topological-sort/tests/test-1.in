6
A->B, C
B->D, E, A
C->F
D->C, F
E->D
F->
