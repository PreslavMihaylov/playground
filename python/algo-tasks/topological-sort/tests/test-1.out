Topological sort: A, B, E, D, C, F
